doc-Installing_on_OpenShift_Container_Platform/topics/configuring_external_auth_ocp.adoc

```diff
@@ -0,0 +1,278 @@
+
+For usage, run:
+----
+$ httpd_configmap_generator <auth-type> --help
+----
+
+.Supported Authentication Types
+[width="100%",cols="30%,70%,options="header",]
+|====
+|auth-type|Identity Provider/Environment
+| active-directory | Active Directory domain realm join               
+| ipa              | IPA, IPA 2-factor authentication, IPA/AD Trust   
+| ldap             | LDAP directories                                 
+| saml             | Keycloak, Red Hat SSO
+                              
+|====
+
+
+==== Updating an `authconfig` Map
+
+With the `update` subcommand, you can add file(s) to the configuration
+map to specify file ownership and permissions. The `--add-file` option can be specified multiple times (once per file) to add files to a configuration map.
+
+Supported file specifications for the `--add-file` option are:
+
+----
+--add-file=file-path
+--add-file=source-file-path,target-file-path
+--add-file=source-file-path,target-file-path,file-permission
+--add-file=file-url,target-file-path,file-permission
+----
+
+When entering file specifications, `file-url` is an HTTP URL and `file-permission` can be specified as `mode:owner:group`.
+
+
+.Examples
+
+.Adding files by specifying paths
+
+The file ownership and permissions are based on the files specified. For example:
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml                \
+--add-file=/etc/openldap/cacerts/primary-directory-cert.pem  \
+--add-file=/etc/openldap/cacerts/seconday-directory-cert.pem \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+.Adding target files from different source directories
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml                                        \
+--add-file=/tmp/uploaded-cert1,/etc/openldap/cacerts/primary-directory-cert.pem  \
+--add-file=/tmp/uploaded-cert2,/etc/openldap/cacerts/seconday-directory-cert.pem \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+The file ownership and permissions are based on the source files specified;
+in this case the ownership and permissions of the `/tmp/uploaded-cert1`
+and `/tmp/uploaded-cert2` files will be used.
+
+.Adding a target file with user-specified ownership and mode
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml                          \
+--add-file=/tmp/secondary-keytab,/etc/http2.keytab,600:apache:root \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+.Adding files by URL
+
+----
+$ httpd_configmap_generator update \
+--input=/tmp/original-auth-configmap.yaml \
+--add-file=http://aab-keycloak:8080/auth/realms/testrealm/protocol/saml/description,/etc/httpd/saml2/idp-metadata.xml,644:root:root \
+--output=/tmp/updated-auth-configmap.yaml
+----
+
+When downloading a file by URL, a target file path and file ownership/mode must be specified.
```

Comment:
I'd replace the */* with *and* for better readability: "file ownership and mode must be..". What do you think?

Reply:
Good catch. I've made this fix in a new branch, BZ#1540641-addl.